#!/usr/bin/env python
PACKAGE = "hektar"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("speed", int_t, 0, "wheel speed value", 70, 0, 127)
gen.add("variation_factor", double_t, 0, "scalar multiplier for wheel speed", 0.5, 0.0, 1.0)

exit(gen.generate(PACKAGE, "wheel_control", "Hektar"))
